src/main/java/com/shaft/properties/internal/Web.java

```diff
@@ -38,15 +38,15 @@ private static void setProperty(String key, String value) {
 
     @Key("mobileEmulation.width")
     @DefaultValue("")
-    String mobileEmulationWidth();
+    int mobileEmulationWidth();
 
     @Key("mobileEmulation.height")
     @DefaultValue("")
-    String mobileEmulationHeight();
+    int mobileEmulationHeight();
 
     @Key("mobileEmulation.pixelRatio")
-    @DefaultValue("")
-    String mobileEmulationPixelRatio();
+    @DefaultValue("1.0")
+    double mobileEmulationPixelRatio();
```

Comment:
@magdyheiba note that we can use int and double datatypes for parameters instead of using string and later casting it.